rita/build.rs

```diff
@@ -1,9 +1,11 @@
 use std::process::Command;
+
 fn main() {
```

Comment:
why did this end up being changed?

Reply:
Docker images shipped with cross doesn't have git installed. I remember I reported it somewhere in cross repos, but can't find it now. This shouldn't be a problem as long as we're not doing deployments using cross. There is possiblity to specify own images for architectures so we could install binaries. Anyway I think we should be fine.